x/concentrated-liquidity/fees.go

```diff
@@ -185,88 +201,103 @@ func (k Keeper) getInitialFeeGrowthOutsideForTick(ctx sdk.Context, poolId uint64
 	return emptyCoins, nil
 }
 
-// collectFees collects fees from the fee accumulator for the position given by pool id, owner, lower tick and upper tick.
-// Upon successful collection, it bank sends the fees from the pool address to the owner and returns the collected coins.
-// Returns error if:
-// - pool with the given id does not exist
-// - position given by pool id, owner, lower tick and upper tick does not exist
-// - other internal database or math errors.
-func (k Keeper) collectFees(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, lowerTick int64, upperTick int64) (sdk.Coins, error) {
-	feeAccumulator, err := k.getFeeAccumulator(ctx, poolId)
+// collectFees collects the fees earned by a position and sends them to the owner's account.
```

Comment:
The details in the godoc are no longer valid. This is because if the positionId exists, then so does the position itself.

I can add a single line stating "returns error if the provided positionId is invalid", which should be the only error case

Reply:
https://github.com/osmosis-labs/osmosis/pull/4695/commits/fa7edfe4dd7228293de717eec8b76a1ff63e3b22